app/assets/stylesheets/_barcode_card.scss

```diff
@@ -60,8 +60,8 @@
     .btn-icon {
       margin: -3px 0 0;
       font-size: 24px;
-      background: none !important;
       color: inherit;
+      svg { fill: currentcolor; }
```

Comment:
I'm wondering if this shouldn't be the default for *all* SVG icons? I think you would usually want the icons to be in the text color. That was previously the case with the icon font, so it should not be a change. SVG opens more possibilities for coloured icons, but `currentColor` should be the default.

Reply:
See the current [`_sprites.scss`](https://github.com/instedd/cdx/blob/main/app/assets/stylesheets/_sprites.scss#L27), it sets the icon color to black by default.

Reply:
Sigh, it's stupid, but you're right. We should consider changing that some time, though.